interface.go

```diff
@@ -20,3 +21,34 @@ type Session interface {
 	RemoteAddr() net.Addr
 	Close(error) error
 }
+
+// ConnState is the status of the connection
+type ConnState int
+
+const (
+	// ConnStateVersionNegotiated means that version negotiation is complete
+	ConnStateVersionNegotiated ConnState = iota
+	// ConnStateSecure means that the connection is encrypted
+	ConnStateSecure
+	// ConnStateForwardSecure means that the connection is forward secure
+	ConnStateForwardSecure
```

Comment:
I think these need to be directional, i.e. ConnStateSendingSecure, ConnStateReceivingSecure. Otherwise you cannot do 0-RTT properly, right?